images/make_osx_icon.sh

```diff
@@ -13,23 +13,26 @@ PNGCRUSH="$(which pngcrush)"
 SVG_FILE="${SCRIPT_ROOT}/OmegaT.svg"
 [ ! -f "$SVG_FILE" ] && echo "Can't find SVG to convert." && exit 1
 
-SIZES="32 64 128 256"
-# 512px version is taken from website:
-# http://briac.net/omegat-logo/
+SIZES="32 64 128 256 512 1024"
```

Comment:
Is `icon_16x16.png` generated?

Reply:
I just did and pushed it. Originally, I had one created by a pixel artist (the one that's in the repository). But it was too pixelized.